static/scss/profile-image-uploader.scss

```diff
@@ -22,6 +22,6 @@
 
   .photo-active-item {
     width: 100%;
-    height: 500px;
+    height: 300px;
```

Comment:
I support that plan! optimally we'd show it at the size it currently is on a big display, and shrink down only when needed on smaller displays